ddl/ddl.go

```diff
@@ -459,7 +462,12 @@ func (d *ddl) GetID() string {
 	return d.uuid
 }
 
-func checkJobMaxInterval(job *model.Job) time.Duration {
+func (d *ddl) checkJobMaxInterval(job *model.Job) time.Duration {
+	// testCheckJobMaxInterval's value may not be 0 only in tests.
+	if d.testCheckJobMaxInterval != 0 {
+		return d.testCheckJobMaxInterval
```

Comment:
It is dangerous here, change a testCheckJobMaxInterval maybe neglected by the reviewer. How about make the hard code time in line 474 and line 477 to a variable, then we can modify that variable.

Reply:
We don‘t need to care `testCheckJobMaxInterval`, it only can be set by `SetCheckJobMaxInterval`, and `SetCheckJobMaxInterval` is in ddl_test.